### DISCUSSION TITLE
I'm making a cheat client for a video game using CustomTkinter :)

### DISCUSSION THREAD

**Discussion by kelltom** [TomSchimansky/CustomTkinter#281] [Showcase]
Link to the repo: https://github.com/kell90/osrs-bot

The TLDR for the project: it is a single GUI that lets users manage and monitor automation scripts for a video game.

It's a work in progress, but after less than a month of development, it is on track! I'm currently working on more advanced features for the automation library, such as image recognition, OCR and real-time object contouring. Once that's done, I can start writing real bots.

I know cheating in video games is _bad_, but this is mostly just a portfolio piece to hopefully help me get my first job in the industry. 

Thanks to Tom and everyone contributing to CustomTkinter!

------------------------------

_Navigating UI and running an example bot_

https://user-images.githubusercontent.com/44652363/179625863-2438d9be-6055-49d7-b416-dfce12aabe9b.mp4

_Utilizing some of the library functions for moving the mouse and extracting information from the game_

https://user-images.githubusercontent.com/44652363/179625867-38f68c44-fb65-4c9d-8c15-31832c73e9ae.mp4

**Comment by abhaymakes** [comment]
That looks good @kell90 check out mine too, its a dictionary made in CustomTkinter!

Here : https://github.com/TheFallen-Cat/VocabiPy
